DigitalLearningSolutions.Web/Views/TrackingSystem/CourseSetup/ManageCourse/EditCourseDetails.cshtml

```diff
@@ -0,0 +1,192 @@
+                  Password protected
+                </label>
+              </div>
+
+              <div class="@passwordCss" id="@passwordId">
+                <div class="nhsuk-form-group">
+                  <vc:text-input asp-for="@nameof(Model.Password)"
+                                 label="Enter the password for the course"
+                                 populate-with-current-value="true"
+                                 type="text"
+                                 spell-check="false"
+                                 autocomplete=""
+                                 hint-text=""
+                                 css-class="nhsuk-input nhsuk-u-width-one-half" />
+                </div>
+              </div>
+            </div>
+
+            <div class="nhsuk-checkboxes__item">
+              <input class="nhsuk-checkboxes__input"
+                     id="ReceiveNotificationEmails"
+                     name="ReceiveNotificationEmails"
+                     asp-for="ReceiveNotificationEmails"
+                     type="checkbox"
+                     aria-controls="@emailId"
+                     aria-expanded="@(Model.ReceiveNotificationEmails ? "true" : "false")">
+              <label class="nhsuk-label nhsuk-checkboxes__label" for="ReceiveNotificationEmails">
+                Receive course completion notification emails
+              </label>
+            </div>
+
+            <div class="@emailCss" id="@emailId">
+              <div class="nhsuk-form-group">
+                <vc:text-input asp-for="@nameof(Model.NotificationEmails)"
+                               label="Enter an email address for receiving the notifications"
+                               populate-with-current-value="true"
+                               type="text"
+                               spell-check="false"
+                               autocomplete=""
+                               hint-text=""
+                               css-class="nhsuk-input nhsuk-u-width-one-half" />
+              </div>
+            </div>
+
+          </div>
+        </fieldset>
+      </div>
+
+      <h2 class="nhsuk-heading-s nhsuk-u-font-weight-normal">
+        Completion criteria
+      </h2>
+
+      <div class="nhsuk-form-group">
+        <fieldset class="nhsuk-fieldset">
+          <div class="nhsuk-radios nhsuk-radios--conditional">
+
+            @if (Model.PostLearningAssessment) {
+              <div class="nhsuk-radios__item nhsuk-u-margin-bottom-3">
+                <input class="nhsuk-radios__input"
+                       id="PostLearningAssessmentRadio"
+                       name="IsAssessed"
+                       asp-for="IsAssessed"
+                       type="radio"
+                       value="true">
+                <label class="nhsuk-label nhsuk-radios__label" for="PostLearningAssessmentRadio">
+                  Post learning assessment
+                </label>

```

Comment:
Same here, use asp-for

Reply:
For this one it doesn't let me because the PostLearningAssessmentRadio is not a property on the model, just an id I made, since the radio buttons need different ids but have the same asp-for

Reply:
Yeah that's fine then, leave these 2 cases.